### PR TITLE
EUI-408 Phase banner addition

### PR DESCRIPTION
**Pull request by praveenrajendran101**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/app/containers/header/header.component.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
       [navigation]="userNav"
       (navigate)="onNavigate($event)"
     ></app-hmcts-global-header>
-
+    <app-phase-banner type="beta"></app-phase-banner>
```
Comment (fazanki) [inline src/app/containers/header/header.component.html]: why do we need type tag here?

Comment (adnanakgun) [reply src/app/containers/header/header.component.html]: Hey Franco. This is a string value which is displayed in the component. The same in approve orgs.

Comment (fazanki) [reply src/app/containers/header/header.component.html]: I am not sure how this is not erroring tbh shouldn't then this be [type]='beta'

Comment (praveenrajendran101) [reply src/app/containers/header/header.component.html]: Hi Franko, In [type]='beta', beta is a variable and In type='beta' , beta is a string. these are angular data-binding  fundamentals

**Pull request merged**